.node-version

```diff
@@ -0,0 +1 @@
+20.10.0
```

Comment:
現在のLTSは[v20.12.2](https://arc.net/l/quote/nhdmpeig)なので、最新を指定した方が良さそうだと思いました！

Reply:
I don't think it's necessary to consider the patch version, so I'll use v20.12.0.
This is because Node.js updates patches frequently and the cost of keeping up with them is too high.
If you take minor versions into account, you won't have any problems in most cases.

[80464e1](https://github.com/moneyforward/eslint-config-moneyforward/pull/255/commits/80464e1dc856d309effc8737e048e245c08fb60e)